pychunkedgraph/graph/meta.py

```diff
@@ -80,7 +84,20 @@ def custom_data(self):
     def ws_cv(self):
         if self._ws_cv:
             return self._ws_cv
-        self._ws_cv = CloudVolume(self._data_source.WATERSHED)
+
+        try:
+            # try reading a cached info file for distributed workers
+            # useful to avoid md5 errors on high gcs load
+            cache_key = f"{self.graph_config.ID}:cv_info_cached"
+            redis = get_redis_connection()
+            cached_info = json.loads(redis.get(cache_key))
+            self._ws_cv = CloudVolume(self._data_source.WATERSHED, info=cached_info)
+        except Exception:
```

Comment:
this could be better handled. Isnt there a way to check if the key exists in redis?

Reply:
This handles both redis existence and json parsing issues, we want it to work in both cases so if either fails just reads from info file.

Reply:
And if a redis connection is even available.

Reply:
When running locally, for instance.

Reply:
It's used in the Exception clause and that assumes that the failure happened before the definition

Reply:
Yes, but `try` is executed first, so it will already have been defined when it reaches `except`.